Resources/Assets.xcassets/Results.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "Results.png",
```

Comment:
For images, I try and generate them with CoreGraphics so we don't have to embed lots of files and are always pixel perfect regardless of screen density, look at `IPFIconGenerator.m` as an example. I also tend to reuse system icons as much as possible to go for a vanilla look, that way there is less maintenance for OS updates.

Reply:
I have replaced these ones with 3 pdf versions of system icons provided by SFSymbols.
Specifically used this tool to export 3 icons which seemed to match
https://github.com/davedelong/sfsymbols
Vector pdf's will scale to any size and have been around since Xcode 6, so should be backwards compatible to all OS versions supported by iPerf